documentation/book/proc-deploying-cluster-operator-helm-chart.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// assembly-cluster-operator.adoc
+
+[id='deploying-cluster-operator-helm-chart-{context}']
+= Deploying the Cluster Operator using Helm Chart
+
+.Prerequisites
+
+* Helm has to be installed on the local machine.
+* Helm has to be installed in the {ProductPlatformName} cluster.
+
+.Procedure
+
+. Deploy the Cluster Operator using the Helm command line tool:
++
+[source,shell,subs=attributes+]
+helm install {ChartReleaseDownload}
```

Comment:
Change "{ChartReleaseDownload}" to "<_ChartReleaseDownload_>". Replaceable text has to styled in monospace and surrounded by angle brackets.

http://strimzi.io/docs/master/#document-conventions-str

Reply:
This is actually not something the user should replace. This is a value from attributes.,